app/views/articles/show.html.erb

```diff
@@ -31,6 +31,36 @@
     </script>
   <% end %>
 
+<script type="application/ld+json">
+  {
+    "@context": "http://schema.org",
+    "@type": "Article",
+    "mainEntityOfPage": "True",
+    "url": "<%= article_url(@article) %>",
+    "image": "<%= article_social_image_url(@article) %>",
+    "publisher": {
+      "@context": "http://schema.org",
+      "@type": "Organization",
+      "name": "<%= community_qualified_name %>"
+      "logo": {
+        "@context": "http://schema.org",
+        "@type": "ImageObject",
+        "url": "<%= cloudinary(SiteConfig.logo_png, "png") %>",
+        "width": "192",
+        "height": "192"
```

Comment:
by default the logo is 512x512 so we should probably resize it to 192 with `cloudinary(SiteConfig.logo_png, 192)`. The second argument is the width